script.js

```diff
@@ -4,24 +4,43 @@ var game = {
   deck: [],
   players: [],
   hands: [],
+//Populates deck with standard cards
   buildDeck: function(){
-
+    for (var i = 0; i < values.length; i++){
+      for (var i = 0; i < suits.length; i++)
```

Comment:
This is going to throw some errors because you already have a variable called `i`. If you have nested `for` loops, you need to use a different variable for each of them.